src/magic_folder/uploader.py

```diff
@@ -215,7 +215,6 @@ def add_file(self, path):
 
         # add file into the queue
         d = Deferred()
-        self._status.upload_started()
```

Comment:
Does this still want to be here? I guess it depends on if we want to track "something needs to be uploaded" (in which case we want it) vs. "actively uploading something" in which case we may not.

Reply:
It's gone now .. but also might want to think about what "sync'ing" means. In one sense ("we're outdated") then it should be True here, but in another ("we're actively uploading/donwloading") it may be False here still.